spec/active_record_batch_consumer_spec.rb

```diff
@@ -488,24 +491,5 @@ def record_attributes(payload, key)
       end
     end
 
-    describe 'association_list feature for SQLite database' do
```

Comment:
We still need the integration tag because you can't run these tests without a database. There's a long-standing issue to be able to run these tests on CI.